semgrep-core/src/engine/Augmented_range.ml

```diff
@@ -0,0 +1,122 @@
+type range_kind = Plain | Inside | Regexp [@@deriving show]
+
+(* range with metavars *)
+type range_with_mvars = {
```

Comment:
Once we moved this data type to its own module then I would just call it `t` as it is tradition in OCaml.

Reply:
Yep, I agree.
I would call the module maybe Range_with_metavars.ml